bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/BlueGigaBluetoothDevice.java

```diff
@@ -288,6 +288,12 @@ public boolean disableNotifications(BluetoothCharacteristic characteristic) {
         return true;
     }
 
+    @Override
+    public boolean isNotifying(BluetoothCharacteristic characteristic) {
+        // TODO will be implemented in a followup PR
+        return false;
```

Comment:
No need to wait for another PR.
```suggestion
        BlueGigaBluetoothCharacteristic ch = (BlueGigaBluetoothCharacteristic) characteristic;
        return ch.isNotificationEnabled();
```

And while you are add it. Can you also fix line 249? It looks like I forgot to inverse the condition. Thanks.

Reply:
Nice, I just changed it.
That said, I noticed that we have different namings for the same functionality. The existing `BlueGigaBluetoothDevice.isNotificationEnabled()` and the new `BluetoothDevice.isNotifying()`. The library class `BluetoothGattCharacteristic` uses `isNotifying()` too.

I would prefer to use the same names and don't mind either. What do you think?

Reply:
Might as well change it to `isNotifying` as well.